src/model/kube/statusFetcher.go

```diff
@@ -443,7 +445,16 @@ func checkPublicHealth(ingresses []K8sIngressInfo, healtcheckPath string) bool {
 
 func checkHealth(status AppDeploymentInfo, checkBaseUrl string, healtcheckPath string) (bool, string, string) {
 	checkUrl := checkBaseUrl + healtcheckPath
-	r, httpErr := http.Get(checkUrl)
+
+	client := &http.Client{}
+
+	req, reqErr := http.NewRequest("GET", checkUrl, nil)
+	if reqErr != nil {
+		return false, reqErr.Error(), HealthCheckType_NotCheckedYet
+	}
+
+	req.Header.Add("user-agent", healthCheckUserAgent)
```

Comment:
I'd use `set` in favor of `add` and also use the canonical header key:

```suggestion
	req.Header.Set("User-Agent", healthCheckUserAgent)
```